riscv/rocc.h

```diff
@@ -24,6 +24,7 @@ union rocc_insn_union_t
 class rocc_t : public extension_t
 {
  public:
+  rocc_t(bool always_dirty) : extension_t(always_dirty) {}
```

Comment:
Since `rocc_t` never calls `set_state()` it seems that the code assumes `always_dirty` and hence this should not be a constructor parameter. Instead:
```
rocc_t(): extension_t(true) {}
```